book/forms.rst

```diff
@@ -667,7 +674,7 @@ boxes. However, the :doc:`date field </reference/forms/types/date>` can be
 configured to be rendered as a single text box (where the user would enter
 the date as a string in the box)::
 
-    ->add('dueDate', 'date', array('widget' => 'single_text'))
+    ->add('dueDate', Type\DateType::class, array('widget' => 'single_text'))
```

Comment:
i would prefer to use line breaks in any ``add()`` mehtod as it is more readable